modules/email-chat/email-chat.model.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+
+const EmailChatSchema = new mongoose.Schema({
+  text: {
+    type: String,
+    required: true,
+  },
+  name: {
```

Comment:
Maybe a more descriptive name, like `senderName` to make it less ambiguous?

Reply:
renamed